products/libllbuild/CMakeLists.txt

```diff
@@ -22,15 +22,30 @@ set_property(TARGET libllbuild PROPERTY MACOSX_RPATH ON)
 
 target_link_libraries(libllbuild ${DEPENDENCIES})
 
+# Load version number from the XCConfig
+file(STRINGS "${CMAKE_SOURCE_DIR}/Xcode/Configs/Version.xcconfig" VERSION_XCCONFIG)
+string(REGEX REPLACE ".*LLBUILD_C_API_VERSION = ([0-9]+).*" "\\1" LLBUILD_C_API_VERSION "${VERSION_XCCONFIG}")
```

Comment:
Can we see if we can make CMake error out of this regex fails to match properly? If its non-trivial we can skip it.